tests/handlers/test_queries.py

```diff
@@ -185,6 +187,27 @@ def test_create_query(self):
         self.assertTrue(query.is_draft)
 
 
+class TestQueryArchiveResourceGet(BaseTestCase):
+    def test_returns_queries(self):
+        q1 = self.factory.create_query(is_archived=True)
+        q2 = self.factory.create_query(is_archived=True)
+        q3 = self.factory.create_query()
+
+        rv = self.make_request('get', '/api/queries/archive')
+
+        assert len(rv.json['results']) == 2
+        assert set(map(lambda d: d['id'], rv.json['results'])) == set([q1.id, q2.id])
+
+    def test_search_term(self):
+        q1 = self.factory.create_query(name="Sales", is_archived=True)
+        q2 = self.factory.create_query(name="Q1 sales", is_archived=True)
+        q3 = self.factory.create_query(name="Q2 sales")
```

Comment:
nit: In this test and the above one, you create `q3` but never reference it. Maybe it doesn't need to be placed in the variable `q3` if it won't be used or maybe come additional assert can be done to confirm that it's not returned in the request?

Reply:
Totally!